src/database/migrations/0035-alter-proposal_versions-add-created_by.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE proposal_versions
+  ADD COLUMN created_by INTEGER NOT NULL DEFAULT select_system_user_id(),
+  ADD CONSTRAINT fk_createdBy
```

Comment:
PostgreSQL will lower-case this unquoted mixed-case [identifier](https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS). I think you want it to be snake_cased.